
point1:
  position:
    x: -1.34904427196
    y: 5.76954035449
    z: 0.0
  orientation:
    x: 0.0
    y: 0.0
    z: 0.701782779552
    w: 0.712390995398

point2:
  position:
    x: -0.830447338086
    y: -3.070927831
    z: 0.0
  orientation:
    x: 0.0
    y: 0.0
    z: -0.521059049727
    w: 0.853520630505